graphql/models/City.graphql

```diff
@@ -0,0 +1,24 @@
+type Query{
+    cities:[City!]! @paginate(defaultCount:10)
+    city(id:ID @eq):City @find
+}
+
+type City{
+    id: ID!
+    name : String!
+    area : String!
+    gover_id : String!
+    is_division : String!
+    districts : String!
+    divisions : String!
+    units : String!
+    population : String!
+    p_density_rate : String!
```

Comment:
same here camelCase

Reply:
Ok I will modify all to CamelCase 
and remove id from all

Reply:
don't remove `id` I mean only removing the foreign id likes `gover_id`